src/gradata/hooks/config_validate.py

```
@@ -0,0 +1,83 @@
+                module_name = command.split("gradata.hooks.")[-1].split()[0].strip('"\'')
+                try:
+                    import gradata.hooks as hooks_pkg
+                    hooks_dir = Path(hooks_pkg.__file__).parent
+                    module_path = hooks_dir / f"{module_name}.py"
+                    if not module_path.is_file():
+                        warnings.append(
+                            f"hooks.{event_name}[{i}] references "
+                            f"gradata.hooks.{module_name} but module not found"
+                        )
+                except Exception:
+                    pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)